### PR TITLE
Focus mode

### PR DESCRIPTION
**Pull request by webofpies**
Focus mode (issue #238)

Padding of the text can be set by the "content width" buttons in the side menu.

image

### CODE REVIEW

File: lute/static/css/styles.css [hunk 1]
```diff
@@ -987,21 +1041,19 @@ textarea {
 }
 
 /* focus on fields makes border same as tags */
-input:focus,
-textarea:focus,
-select:focus {
```
Comment (jzohrab) [inline lute/static/css/styles.css:992]: select:focus no longer needed?

File: lute/static/css/styles.css [hunk 2]
```diff
@@ -987,21 +1041,19 @@ textarea {
...
+form input:focus,
+form textarea:focus {
     border: 1px solid var(--tagify__tag-bg) !important;
     outline: none !important;
 }
 
-input[type="checkbox"]:focus,
-input[type="file"]:focus,
-button:focus {
```
Comment (jzohrab) [inline lute/static/css/styles.css:999]: button:focus no longer needed?

**Comment by jzohrab** [comment]
With all of the hacking for dict tabs recently merged, there are now some conflicts.  Might need some checking after resolving and merging, to ensure that the dict js doesn't throw with these changes.  Otherwise, looks super.

**Review by jzohrab — CHANGES_REQUESTED** [review]
Just need the conflicts resolved.

File: lute/static/css/styles.css (hunk 1 above)
Comment (webofpies) [reply lute/static/css/styles.css:1001]: fixed

File: lute/static/css/styles.css (hunk 2 above)
Comment (webofpies) [reply lute/static/css/styles.css:1008]: focus border on buttons didn't look that nice. removed
also there are some buttons we don't the border or outline for. easier to remove here than add exceptions

**Review by jzohrab — APPROVED** [review]
LGTM 🚀

**Pull request merged**